package.json

```diff
@@ -90,7 +90,6 @@
     "devDependencies": {
         "@types/mocha": "^7.0.2",
         "@types/node": "^13.11.0",
-        "@types/vscode": "^1.44.0",
```

Comment:
I think we should actually remove it from the main dependencies instead? See [discussion](https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies) - I looked at this last week and couldn't make up my mind.